### PR TITLE
Much too creative changes to alcohol

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Turns alcohol into a proper property and adds a limit to the properties, you won't slur for 25 years from a single shot
(I will remove the drink if needed)
# Explain why it's good for the game

It really isnt

# Changelog

:cl:
add: Reworks alcohol and its property and makes it way too complicated
fix: Adds a limit to how long you will be slurring for after drinking alcohol
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry_properties/prop_neutral.dm [hunk 1]
```diff
@@ -135,15 +135,37 @@
...
+		M.sleeping = min(M.sleeping + 0.5 * potency * delta_time, 10 * potency)
+	M.confused = min(M.confused + 2 * potency * delta_time, 10 * potency)
+	M.drowsyness = min(M.drowsyness + 2 * potency * delta_time, 10 * potency)
+	M.dizziness += min(M.dizziness + 0.5 * potency * delta_time, 10 * potency)
+	M.slurring += min(M.slurring + 1 * potency * delta_time, 10 * potency)
+	M.apply_damage(0.5 * potency * delta_time, TOX)
+	M.apply_damage(1 * potency * delta_time, OXY)
+	if(prob(5 * potency * delta_time) && ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.vomit()
```
Comment (Zonespace27) [inline code/modules/reagents/chemistry_properties/prop_neutral.dm:168]: this is all... completely unreadable
Please space it out (also no one letter variables)
Turning some of the numbers into defines wouldn't hurt

File: code/modules/reagents/chemistry_reagents/alcohol.dm [hunk 2]
```diff
@@ -18,15 +18,13 @@
...
 	var/boozepwr = 5 //higher numbers mean the booze will have an effect faster.
-	var/booze_data = 0
-	var/dizzy_adj = 3
-	var/adj_drowsy = 0
-	var/adj_sleepy = 0
-	var/slurr_adj = 3
-	var/confused_adj = 2
-	var/slur_start = 300 //(formerly 180) amount absorbed after which mob starts slurring
+	var/booze_data = 0 //If it's a normal drink, let them drink a bit before they start feeling it.
+	var/slur_start = 250 //This is when the properties are gonna hit. Drink in moderation.
```
Comment (Zonespace27) [inline code/modules/reagents/chemistry_reagents/alcohol.dm:26]: code doc

File: code/modules/reagents/chemistry_reagents/alcohol.dm [hunk 3]
```diff
@@ -18,15 +18,13 @@
...
+	overdose_critical = HIGH_REAGENTS_OVERDOSE_CRITICAL
 	var/boozepwr = 5 //higher numbers mean the booze will have an effect faster.
-	var/booze_data = 0
-	var/dizzy_adj = 3
-	var/adj_drowsy = 0
-	var/adj_sleepy = 0
-	var/slurr_adj = 3
-	var/confused_adj = 2
-	var/slur_start = 300 //(formerly 180) amount absorbed after which mob starts slurring
+	var/booze_data = 0 //If it's a normal drink, let them drink a bit before they start feeling it.
```
Comment (Zonespace27) [inline code/modules/reagents/chemistry_reagents/alcohol.dm:25]: Is this used anywhere anymore?

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]
i looove properties wow!!

**Pull request merged**